solidity/contracts/OngoingAirdrops.sol

```diff
@@ -15,7 +15,7 @@ contract OngoingAirdrops is AccessControl, IOngoingAirdrops {
   /// @inheritdoc IOngoingAirdrops
   mapping(bytes32 => bytes32) public roots;
   /// @inheritdoc IOngoingAirdrops
-  mapping(bytes32 => uint256) public amountClaimedByCampaignTokenAndUser;
+  mapping(bytes32 => uint256) public amountClaimedByCampaignTokenAndClaimee;
```

Comment:
You are sometimes using `user` and sometimes using `claimee`. I'm ok either way, but could we use the same everywhere?